app/presenters/renewal_letter_presenter.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class RenewalLetterPresenter < BaseLetterPresenter
+  # Provides the full postal address for the letter.
+  def postal_address_lines
+    lines = [contact_full_name]
+    lines << operator_name unless business_type == WasteExemptionsEngine::Registration::BUSINESS_TYPES[:sole_trader]
+    lines << address_lines(contact_address)
+    lines.flatten!.reject(&:blank?)
+  end
+
+  def exemption_description(exemption)
+    "#{exemption.code} #{exemption.summary}"
+  end
+
+  def expiry_date
+    # Currently you can only add exemptions when you register, so we can assume they expire at the same time
+    registration_exemptions.first.expires_on.to_formatted_s(:day_month_year)
+  end
+
+  def listable_exemptions
+    @_listable_exemptions ||= exemptions.first(18)
+    @_listable_exemptions
+  end
+
+  def unlisted_exemption_count
+    @_unlisted_exemption_count ||= calculate_number_of_unlisted_exemptions
+  end
+
+  def partners_list
+    return unless partnership?
+
+    people_list = []
+    people.each do |person|
+      people_list << format_names(person.first_name, person.last_name)
+    end
+
+    people_list.join(", ")
+  end
+
+  def site_description
+    if site_description_abridged?
+      site_address.description.truncate(max_site_description_length, separator: " ")
+    else
+      site_address.description
+    end
+  end
+
+  def site_description_abridged?
+    @_site_description_abridged ||= site_address.description.length > max_site_description_length
+  end
+
+  private
+
+  def max_site_description_length
```

Comment:
I think this is a good candidate for a class constant rather than a private method, WDYT?

Reply:
Resolved by https://github.com/DEFRA/waste-exemptions-back-office-ta/pull/357/commits/f156d14ad5cd44ad88c76ac4a9da4000480091b2